troposphere/static/js/components/images/detail/header/HeaderView.react.js

```diff
@@ -1,14 +1,15 @@
 
 define(
   [
-    'react',
+    'react/addons',
     'backbone',
     'jquery',
     'stores',
+    'modals',
     // plugins
-    'bootstrap'
+    // 'bootstrap' // Not used?
```

Comment:
More occurrences of this: 
```javascript
  // plugin: required but not used directly
  bootstrap = require('bootstrap');
```
- [ButtonGroup.react](https://github.com/iPlantCollaborativeOpenSource/troposphere/blame/master/troposphere/static/js/components/common/ButtonGroup.react.js#L5-L6)
- [InteractiveDateField.react](https://github.com/iPlantCollaborativeOpenSource/troposphere/blame/master/troposphere/static/js/components/common/InteractiveDateField.react.js#L7-L8)
- [Tag.react](https://github.com/iPlantCollaborativeOpenSource/troposphere/blame/master/troposphere/static/js/components/common/tags/Tag.react.js#L7-L9)
- [RequestResourcesButton.react](https://github.com/iPlantCollaborativeOpenSource/troposphere/blame/master/troposphere/static/js/components/projects/detail/resources/RequestResourcesButton.react.js#L9-L10)
- [RefreshButton.react](https://github.com/iPlantCollaborativeOpenSource/troposphere/blame/master/troposphere/static/js/components/projects/detail/resources/RefreshButton.react.js#L5-L9)
- [Button.react](https://github.com/iPlantCollaborativeOpenSource/troposphere/blame/master/troposphere/static/js/components/projects/detail/resources/Button.react.js#L5-L6)

Nearly all of the base occurrences stem from a refactoring commit (54e152f236fb02939c9413d9a4b0486e1de70cb2 <= if the link doesn't load, try again). 


This was committed by Jason: 
- [ProjectResource.react](https://github.com/iPlantCollaborativeOpenSource/troposphere/blame/master/troposphere/static/js/components/projects/list/ProjectResource.react.js#L6-L8

Maybe it was just carried forward as a _copy-pasta_ after that? @prosif - @steve-gregory